gateway-release/pom.xml

```diff
@@ -439,7 +439,7 @@
         </dependency>
         <dependency>
             <groupId>org.apache.knox</groupId>
-            <artifactId>gateway-cm-integration</artifactId>
+            <artifactId>gateway-topology-hadoopxml</artifactId>
```

Comment:
nit: since hadoop and xml are two different words do you think the name `gateway-topology-hadoop-xml` is more appropriate? it does look a bit long and awkward though.

Reply:
Fine by me.